notification/webhook/sender.go

```diff
@@ -97,6 +107,7 @@ var _ nfydest.MessageSender = &Sender{}
 func (s *Sender) SendMessage(ctx context.Context, msg notification.Message) (*notification.SentMessage, error) {
 	cfg := config.FromContext(ctx)
 	var payload interface{}
+	pubURL := strings.TrimRight(cfg.General.PublicURL, "/")
```

Comment:
Use `cfg.PublicURL()` (or `cfg.CallbackURL(...)`) as `cfg.General.PublicURL` is deprecated.

```suggestion
	pubURL := cfg.PublicURL()
```